interface-elements-for-desktop/articles/pdf-viewer.md

```diff
@@ -4,13 +4,15 @@ author: Alexander Temnov
 legacyId: 17451
 ---
 # PDF Viewer
-This section describes the capabilities of the **PDF Viewer**.
 
-The PDF Viewer used to displaying PDF files and supports various document content such as text, images, vector graphics, etc.
+This section describes the **PDF Viewer** capabilities.
+
+The PDF Viewer is used to display PDF files and supports various document content such as text, images, vector graphics, etc.
 
 ![pdf-viewer-control-windows-forms](../images/img19650.jpg)
 
-The following sections are available in this guide.
+The following sections are available:
```

Comment:
Why do you say that sections are available? Do you mean this topic consists of these sections?